lib/rubocop/cop/util.rb

```diff
@@ -148,6 +148,10 @@ def move_pos(src, pos, step, condition, regexp)
         pos
       end
 
+      def looking_at(range, regexp)
+        range.source_buffer.source.match(/\G#{regexp}/, range.begin_pos)
```

Comment:
The documentation at http://ruby-doc.org/core-1.9.3/Regexp.html (looking at the oldest Ruby we support) says "\G - Matches point where last match finished", so I couldn't understand what *last match* meant in this case.

I think we need a source comment here, since it's difficult to look up what `\G` does. The name `looking_at` is good, though. Perhaps `looking_at?` is even better if we're going to use it as a Boolean.